package.json

```diff
@@ -3,16 +3,12 @@
   "version": "0.0.19",
   "license": "MIT",
   "description": "Regular Expressions parser in JavaScript",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/DmitrySoshnikov/regexp-tree.git"
-  },
+  "repository": "DmitrySoshnikov/regexp-tree",
   "homepage": "https://github.com/DmitrySoshnikov/regexp-tree",
   "bugs": "https://github.com/DmitrySoshnikov/regexp-tree/issues",
-  "main": "index.js",
   "scripts": {
     "build": "./scripts/build",
-    "test": "./scripts/test",
+    "test": "jest",
```

Comment:
Yep, `npm` prepends ([docs](https://docs.npmjs.com/misc/scripts#path)) `node_modules/.bin` to `PATH` before running the code, so it does not require `jest` installed globally. Travis passed, and it does not have global `jest`.